Add new AuditLogAction types (#2427)
Rapptz/discord.py: docs/api.rst

        - :attr:`~AuditLogDiff.roles`

    .. attribute:: member_move

        A member's voice channel has beenn updated. This triggers when a
        member is moved to a different voice channel.

>         When this is the action, the type of :attr:`~AuditLogEntry.target` is

Reviewer: No: discordapp/discord-api-docs#1195

        the :class:`Member` or :class:`User` which was moved.

    .. attribute:: member_disconnect

        A member's voice state has changed. This triggers when a
        member is force disconnected from voice.

>         When this is the action, the type of :attr:`~AuditLogEntry.target` is

Reviewer: No again: discordapp/discord-api-docs#1195

        - :attr:`~AuditLogDiff.roles`

    .. attribute:: member_move

>         A member's voice channel has beenn updated. This triggers when a

Reviewer 2: ```suggestion
        A member's voice channel has been updated. This triggers when a
```

        member is moved to a different voice channel.

        When this is the action, the type of :attr:`~AuditLogEntry.extra` is
        set to an unspecified proxy object with two attributes:

>         - ``channel``: A :class:`TextChannel` or :class:`Object` with the channel ID of which the members were moved to.

Reviewer 2: Perhaps?
```suggestion
        - ``channel``: A :class:`TextChannel` or :class:`Object` which corresponds to the channel where the member was moved.
```

Reviewer 2: Or actually, following the existing style:
```suggestion
        - ``channel``: A :class:`TextChannel` or :class:`Object` with the channel ID where the members were moved.
```
